crates/deltalake-core/src/operations/transaction/protocol.rs

```diff
@@ -142,6 +142,25 @@ impl ProtocolChecker {
             })?;
         }
 
+        // Does the table have invariants?
+        let invariants = snapshot
+            .current_metadata()
+            .and_then(|meta| meta.schema.get_invariants().ok())
+            .unwrap_or_default()
+            .is_empty();
```

Comment:
`invariants = false` meaning "it does have invariants" does feel very intuitive. Could we reverse it, and rename so it's a little clearer?

```suggestion
        let has_invariants = !snapshot
            .current_metadata()
            .and_then(|meta| meta.schema.get_invariants().ok())
            .unwrap_or_default()
            .is_empty();
```

Reply:
done